volatility3/framework/plugins/windows/hashdump.py

```diff
@@ -21,7 +21,7 @@ class Hashdump(interfaces.plugins.PluginInterface):
     """Dumps user hashes from memory"""
 
     _required_framework_version = (2, 0, 0)
-    _version = (1, 1, 0)
```

Comment:
Why is this one getting bumped?  It should also be a PATCH bump (but please do bump the patch versions, for all the ones that change, so we can tell if the change we made goes wrong).

Reply:
Ok this is just patch bump now for hashdump.py

Reply:
I mean... you can but there's no need.  Dependencies changing should require a bump of the plugin version (because the plugin won't necessarily know it externals have changed), but as I say, not a problem, better to bump then all than none of them...  5:)